tests/baselines/reference/tsConfig/Default initialized TSConfig/tsconfig.json

```diff
@@ -1,66 +1,69 @@
 {
   "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig.json to read more about this file */
+
     /* Basic Options */
-    // "incremental": true,                   /* Enable incremental compilation */
```

Comment:
Example shows the comments as not removed https://github.com/microsoft/TypeScript/issues/37361#issuecomment-598867169 @RyanCavanaugh @DanielRosenwasser Whats the final call on that